package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "gitcoin",
+  "version": "1.0.0",
+  "description": "<img src='https://d3vv6lp55qjaqc.cloudfront.net/items/263e3q1M2Y2r3L1X3c2y/helmet.png'/>",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "lint": "eslint app/static/v2/js/**/*.js",
+    "build": "webpack --config webpack.config.js --progress --colors",
+    "build-production": "webpack --config webpack.prod.config.js --progress --colors",
+    "watch": "node server.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ethikz/web.git"
+  },
+  "author": "Josh Christensen http://github.com/ethikz",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/ethikz/web/issues"
```

Comment:
Update the properties to point to gitcoin:
- `repository.url`
- `author`
- `bugs.url`

Also description, do you wanna go with the image or something like ` "description": "monetize or incentivize work in OSS"` ? Just a thought (It never occurred we could use an image url in the description field ^_^ )

Reply:
Yea good point.  Oy I need to pay more attention lol

We can use the description that Gitcoin uses as slogan I suppose.
- No you're right we don't need one yet.  Eventually we will, or at least I think we will.  One reason to use es6 is importing files so I think if we can have one entry point for the JS that would be good.  Maybe I'm overthinking it?

Reply:
IMHO we could put it off until we define the entry point (will happen eventually ^_^ )